packages/@okta/vuepress-theme-prose/assets/css/layouts/_layout.scss

```diff
@@ -4,6 +4,7 @@ body {
   font-weight: 400;
   line-height: 1.5;
   font-feature-settings: "lnum","pnum";
+  background-color: #16262f;
```

Comment:
The error pages are much shorter and the `body` background color (white) leaks out at the bottom when the viewport is taller than the content, which looks bad.

Since we launched the redesign, we can safely change the global `body` tag style.

Reply:
This will break the docs pages

Reply:
![Uploading Screen Shot 2021-01-26 at 10.21.55 AM.png…]()

Reply:
![Screen Shot 2021-01-26 at 10 21 55 AM](https://user-images.githubusercontent.com/72201855/105864953-72d73c80-5fc0-11eb-9e15-18bf8a1bdd9e.png)

Reply:
Yeah, good call - I added another CSS rule for the page-body to set it back to white. Should play fine since the pages that then add a darker bg use yet another nested div to set that. Deployed to netlify to verify on existing pages but I've clicked around a bit and didn't find it causing any issues.